src/Enigma2.cpp

```diff
@@ -1387,6 +1400,19 @@ PVR_ERROR Enigma2::GetEPGForChannel(ADDON_HANDLE handle, const PVR_CHANNEL &chan
       entry.SetPlotOutline("");
     }    
 
+    if (XMLUtils::GetString(pNode, "e2eventgenre", strTmp))
```

Comment:
I'd move this somewhere so you wouldn't have to duplicate the code

Reply:
In fact that whole XML reading block is pretty much duplicated. Which is the focus of my next PR ;)

So will leave it as for the moment if you don't mind?

Reply:
Sure